src/System.Private.CoreLib/src/System/Reflection/LocalVariableInfo.cs

```diff
@@ -1,40 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics;
-
-namespace System.Reflection
-{
-    public class LocalVariableInfo
-    {
-        #region Private Data Members
-        private RuntimeType m_type;
```

Comment:
The runtime depends on the exact names of these fields (look at `src\vm\mscorlib.h`). You cannot make them auto-implemented properties.

Reply:
If you would like to clean this up a bit, you can change the names to have `_` prefix instead of `m_` prefix.

Reply:
So that means we cant move this class to shared ?

Reply:
We can move this class to shared. The shared copy needs to be more like what is in CoreCLR, and less like what is in CoreRT.

Reply:
@jkotas Out of curiosity, does order also matter or simply the name? I am assuming it does matter.

Reply:
The order has to match https://github.com/dotnet/coreclr/blob/fcb04373e2015ae12b55f33fdd0dd4580110db98/src/vm/runtimehandles.h#L102

Reply:
BTW: it should be fine to change isPinned field to bool (on both managed and unmamaged sides) to make this look better.